### PR TITLE
tests: install specific versions of test package to avoid test packages update breaks

### PR DESCRIPTION
**Pull request by vrutkovs**
Signed-off-by: Vadim Rutkovsky <vrutkovs@redhat.com>

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -78,8 +76,7 @@ $RUN $PYTHON setup.py install
 
 # Install packages for tests
 $RUN $PIP install -r tests/requirements.txt
+if [[ $OS != "fedora" ]]; then $RUN $PIP install -U setuptools; fi
```
Comment (lcarva) [inline test.sh]: Why this line?

Comment (vrutkovs) [reply test.sh]: CentOS has some ancient setuptools installed, which [breaks](https://stackoverflow.com/questions/26589990/py-test-error-unrecognized-arguments-cov-ner-brands-cov-report-term-missi/41066450#41066450) `pytest` and `pytest-cov` integration.

Comment (lcarva) [reply test.sh]: Let's add a comment then, please.

Comment (vrutkovs) [reply test.sh]: Right, done

**Pull request merged**